Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2555,16 +2532,31 @@ def long_running_main():
     Starts the long running thread.
     """
     try:
-        asyncio.run(start_listening(), debug=True)
+        start_listening()
     except Exception as e:
         demisto.error(f"The Loop has failed to run {str(e)}")
-    finally:
-        loop = asyncio.get_running_loop()
+
+
+def fetch_incidents():
+    """
+    Used to offload processing into a background task
+    This will run every 15 seconds. For fetch intervals of 1 minute, this should execute 4 times.
+    """
+    interval = 0
+    while interval < 3:
```

Comment:
since fetch is running every minute, but not necessarily in an perfect accurate timing, this sold be a probably 10-12 second so we will make sure it will end before the next fetch needs to run.
also, why did you pick 15 sec in general?